### PR TITLE
Final MileStone

### PR DESCRIPTION
**Pull request by Said-laasri**
**For this Capstone Project:**

 **Day 1**
- Selected the Animal Zoo API.
- Built file structure for my React app.
- Prepared routes and navigation for my app.

**Day 2**
- display a list of items and filter them. Data come from the API.
- Created the tests for the application.

**Day 3**
- Styled my components to match the design provided.
- Deployed the project and tested for final details.
- Recorded a video for my project.
- Created a good README

Loom Video Link: https://www.loom.com/share/b363c1aa0c3e4a39b77faac5188c83e9

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import MoreAnimalInfo from './component/MoreAnimalInfo';
+
+function App() {
+  return (
+    <div className="App">
+      <Header />
+      <Routes>
+        <Route path="/more/:moreId" element={<MoreAnimalInfo />} />
+        <Route path="/" element={<Animals />} />
+      </Routes>
```
Comment (Wes-Isaac) [inline src/App.js:14]: - [x] Great job so far, one of the requirements is to add a way (component) to filter the presented data on the front page. Kindly add that.

Screen Shot 2022-09-01 at 12 04 10 PM

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @Said-laasri, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
- [x] Please remove the unused files that are generated when you created the project with create react app(CRA), for example [reportWebVitals](https://github.com/Said-laasri/Animal-Zoo/blob/dev/src/reportWebVitals.js)

_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/App.js (hunk 1 above)
Comment (Said-laasri) [reply src/App.js:14]: @Wes-Isaac  thanks for review i created a list of type of animals

**Review by Wes-Isaac — APPROVED** [review]
Hi @Said-laasri, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by Said-laasri** [comment]
@Wes-Isaac  Thank you for reviewing my project.

**Pull request merged**